app/controllers/buyers/applications_controller.rb

```diff
@@ -12,7 +12,8 @@ class Buyers::ApplicationsController < FrontendController
   before_action :find_cinstance, :except => [:index, :create, :new]
   before_action :find_provider,  only: %i[new create update]
 
-  before_action :find_application_plan,          :only => :create
+  before_action :find_accessible_services,
+                :find_accessible_plans,    only: %i[index new create change_plan]
```

Comment:
Why not for `update` and `edit`?
![image](https://user-images.githubusercontent.com/11318903/78290051-69f21780-7523-11ea-8851-c41414aa99a8.png)

Btw, no idea what these `bulk` actions are and if they affect us (if we should apply that restriction for those or not):
![image](https://user-images.githubusercontent.com/11318903/78290108-7f674180-7523-11ea-9727-ffd7b32890dd.png)

Reply:
> update & edit
It's not possible to edit the plan_id; it needs to go tru `change_plan`.

> bulk actions
Not sure about this but I'd say it also goes thru `change_plan`

Reply:
Awesome, makes sense 👍